node/src/components/block_proposer/event.rs

```diff
@@ -72,7 +72,7 @@ impl DeployType {
     }
 
     /// Asks if the variant is Wasm.
-    pub fn is_wasm(&self) -> bool {
+    pub fn is_other(&self) -> bool {
```

Comment:
`is_wasm` was a better name IMO. I understand that the Wasm variants is called `Other` right now but in our domain it's either Wasm or native transfer and I would stick to these names for now. When we get to add other variants w can change names, add other methods.

Reply:
Fair enough, I'll change it back. :+1: